src/properties.jl

```diff
@@ -210,6 +218,10 @@ end
 # # differentiation - cf #27
 # *(φ::SphericalVector, dAA::SVector) = φ.val * dAA'
 
+isrealB(::SphericalVector) = false 
+isrealAA(::SphericalVector) = false 
+
```

Comment:
@zhanglw0521 please check that you agree

Reply:
also also below

Reply:
Thanks for the rewriting. I see no mistake both here and below.